docs/embeddings/hugging-face-embedding-server.md

```diff
@@ -26,7 +26,7 @@ docker compose -f examples/server_side_embeddings/huggingface/docker-compose.yml
 or
 
 ```bash
-docker run -p 8001:80 -d -rm --name huggingface-embedding-server ghcr.io/huggingface/text-embeddings-inference:cpu-0.3.0 --model-id BAAI/bge-small-en-v1.5 --revision -main
+docker run -p 8001:80 -d --rm --name huggingface-embedding-server ghcr.io/huggingface/text-embeddings-inference:cpu-0.3.0 --model-id BAAI/bge-small-en-v1.5 --revision -main
```

Comment:
@mohdwasim, thanks for this. Can we instead use the following (from the latest version of hf repo)

```bash
docker run --platform linux/amd64 -d --rm -p 8080:80 -v ./data:/data --pull always ghcr.io/huggingface/text-embeddings-inference:cpu-1.2 --model-id "BAAI/bge-large-en-v1.5" --revision "refs/pr/5"
```

The reason we add `--platform` is so that it works on MacOS. Also we add a mount to store the model so we don't have to redownload.